OSError: config file not found in examples/standalone_dns

I am trying to follow the [quickstart guide]https://vaping.readthedocs.io/en/stable/quickstart/) but getting the following error:

```
$ vaping start --home examples/standalone_dns 
Traceback (most recent call last):
  File "/home/casadmin/.pyenv/versions/3.9.6/bin/vaping", line 8, in <module>
    sys.exit(cli())
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/click/decorators.py", line 84, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/vaping/cli.py", line 72, in start
    update_context(ctx, kwargs)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/vaping/cli.py", line 21, in update_context
    ctx.update_options(kwargs)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/munge/click.py", line 119, in update_options
    self.config = self.config_class(read=self.home)
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/munge/config.py", line 54, in __init__
    self.read(kwargs["read"])
  File "/home/casadmin/.pyenv/versions/3.9.6/lib/python3.9/site-packages/munge/config.py", line 133, in read
    raise OSError(f"config file not found in {conf_path}")
OSError: config file not found in examples/standalone_dns
```

```
$ vaping --version
vaping, version 1.4.1
$ python --version
Python 3.9.6
$ pip list
Package            Version
------------------ ---------
certifi            2021.5.30
charset-normalizer 2.0.3
click              8.0.1
confu              1.8.0
docutils           0.17.1
Flask              1.1.2
graphsrv           2.0.0
idna               3.2
itsdangerous       2.0.1
Jinja2             2.11.3
lockfile           0.12.2
MarkupSafe         2.0.1
munge              1.2.0
pid                3.0.4
pip                21.1.3
pluginmgr          1.0.1
python-daemon      2.3.0
requests           2.26.0
setuptools         56.0.0
tmpl               1.0.0
urllib3            1.26.6
vaping             1.4.1
vodka              3.1.0
Werkzeug           2.0.1
```

For sure, here you go. 

```
$ gunicorn -b 127.0.0.1:7021 vodka.runners.wsgi:application
[2021-07-20 18:41:03 +0000] [11579] [INFO] Starting gunicorn 20.1.0
[2021-07-20 18:41:03 +0000] [11579] [INFO] Listening at: http://127.0.0.1:7021 (11579)
[2021-07-20 18:41:03 +0000] [11579] [INFO] Using worker: sync
[2021-07-20 18:41:03 +0000] [11580] [INFO] Booting worker with pid: 11580
[2021-07-20 18:41:03 +0000] [11580] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib64/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/vodka/runners/wsgi.py", line 11, in <module>
    config.read(config_dir=os.environ.get("VODKA_HOME", "."))
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/vodka/config/__init__.py", line 420, in read
    return super().read(config_dir=config_dir, clear=clear)
  File "/home/zbiles/customer_jobs/northern_wireless/vaping/lib64/python3.9/site-packages/munge/config.py", line 128, in read
    raise OSError(f"config dir not found at {conf_path}")
OSError: config dir not found at examples/standalone_dns/vodka
[2021-07-20 18:41:03 +0000] [11580] [INFO] Worker exiting (pid: 11580)
[2021-07-20 18:41:03 +0000] [11579] [INFO] Shutting down: Master
[2021-07-20 18:41:03 +0000] [11579] [INFO] Reason: Worker failed to boot.
```

```
$ vaping --version
vaping, version 1.4.1
$ python --version
Python 3.9.2
$ pip list
Package            Version
------------------ ---------
certifi            2021.5.30
charset-normalizer 2.0.3
click              8.0.1
confu              1.8.0
docutils           0.17.1
Flask              1.1.4
graphsrv           2.0.0
gunicorn           20.1.0
idna               3.2
itsdangerous       1.1.0
Jinja2             2.11.3
lockfile           0.12.2
MarkupSafe         2.0.1
munge              1.2.0
pid                3.0.4
pip                21.1.3
pluginmgr          1.0.1
python-daemon      2.3.0
PyYAML             5.4.1
pyyml              0.0.2
pyzmq              22.1.0
requests           2.26.0
setuptools         50.3.2
tmpl               1.0.0
urllib3            1.26.6
vaping             1.4.1
vodka              3.1.0
Werkzeug           1.0.1
wheel              0.36.2
zmq                0.0.0
```

--- a/CHANGELOG.yaml
+++ b/CHANGELOG.yaml
@@ -1,6 +1,7 @@
 Unreleased:
   added: []
-  fixed: []
+  fixed:
+  - missing config deps (#123)
   changed: []
   deprecated: []
   removed: []

--- a/CHANGELOG.yaml
+++ b/CHANGELOG.yaml
@@ -2,6 +2,7 @@ Unreleased:
   added: []
   fixed:
   - missing config deps (#123)
+  - install the actual package in the container image (#122)
   changed: []
   deprecated: []
   removed: []

--- a/Dockerfile
+++ b/Dockerfile
@@ -81,8 +81,7 @@ RUN env
 # install dev
 RUN apk --update add $build_packages
 RUN pip install "poetry$poetry_pin"
-RUN poetry install -E $vaping_extras
-
+RUN poetry install --no-root
 
 
 # execute from final image

--- a/Dockerfile
+++ b/Dockerfile
@@ -43,7 +43,10 @@ COPY src src
 
 # Need to upgrade pip and wheel within Poetry for all its installs
 RUN poetry run pip install --upgrade pip wheel
-RUN poetry install --no-dev -E $vaping_extras
+# poetry installs a pth for root
+RUN poetry install --no-dev --no-root
+RUN poetry build -f wheel
+RUN pip install "$(ls dist/vaping-*.whl)[all]"
 
 # TODO testing stage in container for package deps, etc
 
@@ -79,6 +82,7 @@ RUN env
 RUN apk --update add $build_packages
 RUN pip install "poetry$poetry_pin"
 RUN poetry install -E $vaping_extras
+
 
 
 # execute from final image

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -36,7 +36,7 @@ python = "^3.6.2"
 pid = "^3"
 pluginmgr = "^1"
 python-daemon= "^2"
-munge = "^1"
+munge = {extras = ["tomlkit", "yaml"], version = "^1.2.0"}
 confu = "^1.7.1"
 
 # plugins

